src/AbstractOperators.jl

```diff
@@ -18,11 +16,12 @@ import LinearAlgebra: mul!
 export LinearOperator,
        NonLinearOperator,
        AbstractOperator
+# from other packages
+export ArrayPartition, mul!
```

Comment:
Why these exports?

Reply:
they are not necessary, but quite practical when you load the package to test stuff.

Reply:
I would remove exporting ArrayPartition for sure, I think the package should export what it defines. Maybe mul! could still be exported

Reply:
👍